src/diffusers/schedulers/__init__.py

```diff
@@ -127,6 +127,8 @@
     except OptionalDependencyNotAvailable:
         from ..utils.dummy_pt_objects import *  # noqa F403
     else:
+        from .deprecated.scheduling_karras_ve import KarrasVeScheduler
+        from .deprecated.scheduling_sde_vp import ScoreSdeVpScheduler
```

Comment:
https://github.com/huggingface/diffusers/actions/runs/6509446074/job/17680967513?pr=5269 tells the following:

```
ModuleNotFoundError: No module named 'diffusers.schedulers.deprecated'
```

To check this I did the following:

* Cloned the PR branch and ran `pip install -e .` in a separate env (ensuring PT installation too). 
* From there in the activate env, I was able to successfully import: `import diffusers.scheduler.deprecated`.

This contradicts the error message we get from the CI.

Reply:
```
from diffusers import StableDiffusionControlNetPipeline
```

Also works as expected.

Reply:
Hmm, this seems weird... I'm on python3.11 locally and a fresh install works for me as well without any errors.